### DISCUSSION TITLE
why was useScrollLock hook removed from v7

### DISCUSSION THREAD

**Discussion by mehimanshupatil** [mantinedev/mantine#4960] [Q&A]
I am using hook library and when i upgraded to v7 i see useScrollLock missing. while i checked change log i didn't find any mention of scroll lock hook removed. so why was it removed?

**Comment by rtivital** [comment]
useScrollLock hook was [deprecated in 6.0](https://v6.mantine.dev/changelog/6-0-0/#other-breaking-changes). It is recommended to use `react-remove-scroll` library instead – it handles more cases correctly.
